modules/claims_api/app/controllers/claims_api/base_disability_compensation_controller.rb

```diff
@@ -68,5 +70,19 @@ def track_526_validation_errors(errors)
         StatsD.increment STATSD_VALIDATION_FAIL_TYPE_KEY, tags: ["key: #{key}"]
       end
     end
+
+    def unprocessable_response
+      {
+        data: {
```

Comment:
This should be an error and not a data field. https://discuss.jsonapi.org/t/clarification-on-errors-during-get-requests-404-and-partial-response/254

Reply:
updated